install/linux/usr/share/odemis/sim/enzel-sim.yaml

```diff
@@ -235,7 +235,7 @@ ENZEL-Sim: {
     },
     metadata: {
         # To change what the "good" focus position is on the simulator
-        # FAV_POS_ACTIVE: {'z': 0.0012},  # focus position at init
+         FAV_POS_ACTIVE: {'z': 0.002},  # focus position at init
```

Comment:
Why changing it? Is it to start out of focus?

Reply:
I chose a value in the middle of focus range, but uncommenting this line is more important than the value. Without it, simcam takes the good focus as the current value -> deactive position.

Reply:
Ok, then please also update the comment above to explain that it's needed for not using the initial value, which is at deactive position.

It works actually fine here, and I wonder a little why this is not necessary on my computer. Might be a timing issue between the time it's referenced, and the time the simcam is instantiated.